### ISSUE TITLE
Problem with the bar

### ISSUE THREAD

**Issue by iglesiasji** [closebox73/applying-theme#11]
![Screenshot from 2023-08-10 19-52-55](https://github.com/closebox73/applying-theme/assets/90854600/384269b8-f2fd-4927-9c55-7f5d897d08d1)
Hi,
First Thanks for the theme! , I’m having the next problem, that I get the black bar and not the transparent divisions, nor the drawings. How can I modify it?

**Comment by closebox73** [comment]
@iglesiasji before that I want to ask first, what DE do you use? Gnome?

**Comment by iglesiasji** [comment]
Hi @closebox73 ,yes i use gnome

**Comment by closebox73** [comment]
@iglesiasji check [how to apply](https://github.com/closebox73/applying-theme), maybe it can be answered.

**Comment by iglesiasji** [comment]
@closebox73 , yes, Modify that option, but now it looks like this. All the information is out of date
![Screenshot from 2023-08-13 11-13-12](https://github.com/closebox73/applying-theme/assets/90854600/3cf41628-ec56-4b21-a529-88cb7d152e03)

**Comment by closebox73** [comment]
@iglesiasji it's always going to be like this when you use gnome or budgie, even now I still haven't found a solution, sorry...
